index.json

```diff
@@ -6056,6 +6058,7 @@
   "jungkamushukum.com",
   "junk.beats.org",
   "junk1e.com",
+  "junkmail.com",
```

Comment:
This appears to be a free/paid email service. I wasn't able to find a disposable email service with this domain.

Reply:
My bad, now I find as free email.